exchanges/retry/src/retryExchange.ts

```diff
@@ -75,14 +75,23 @@ export const retryExchange = ({
           })
         );
 
+        dispatchDebug({
+          type: 'retryRetrying',
```

Comment:
Are we making it a standard to prefix it with the exchange or are we going to aim at very generic ones, so it's easier to integrate?

Reply:
Yeah this should have been amended now that we've added the `source` exchange to events.

Multiple exchanges can use the same type, as long as the semantic meaning and passed arguments are the same.